pex/cli/commands/lock.py

```diff
@@ -545,7 +545,7 @@ def _export(self):
                     "{project_name}=={version} \\\n"
                     "  {hashes}\n".format(
                         project_name=pin.project_name,
```

Comment:
Are you sure normalized project names aren't also a problem for pip-tools comparisons? Circling back is fine too, but it would be nice to settle all the pip-tools driven items in 1 release if possible.

Reply:
If I read the code correctly they do "canonicalize" the requirement name: https://github.com/jazzband/pip-tools/blob/a524d983def05802d98e70e08c75c743894562bf/piptools/utils.py#L112 (which "normalizes" it. Good name y'all).